packaging/dbscripts/common_sp.sql

```diff
@@ -55,18 +55,44 @@ LANGUAGE plpgsql;
 CREATE OR REPLACE FUNCTION fn_db_change_column_null (
     v_table VARCHAR(128),
     v_column VARCHAR(128),
-    v_allow_null BOOLEAN
+    v_allow_null BOOLEAN,
+    v_type varchar
```

Comment:
Just an idea: wouldn't it be more flexible to allow pass requested default value as a parameter instead of passing type and having a hardcoded logic for the default value of each type?

Reply:
I don't think so , if you do that you will need to examine the type anyway in order to know if you to add quotes or not when you update the value 
however , I can add an additional optional parameter that if is set than it will be taken instead of the default per type , is that OK?

Reply:
OK, so let's leave as is